frontend/projects/upgrade/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -44,6 +47,7 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
   copyGraphData: IEnrollmentStatByDate[] = [];
   isInitialLoad = true;
   showLabelOfxAxis = true;
```

Comment:
Can you please add the boolean datatype for this?

Reply:
It gave  @typescript-eslint/no-inferrable-types error. So boolean type not assigned.